doc/rfc/0000-rfc-repository.md

```diff
@@ -0,0 +1,168 @@
+and project director.
+
+This makes it difficult (or impossible) for contributors to find documentation
+of what decisions have been made and why. Adopting more open processes for
+project governance will help the OpenTitan project be an exemplary case for
+open hardware development.
+
+## Explanation
+
+RFCs can be very important documentation artefacts. They show exactly what
+decisions have been made and the factors used to come to those decisions. As an
+open source project, we can make this process and the aftefacts more open. This
+RFC proposes applying our open source development model to the RFC process.
+
+We would do this by having RFCs:
+
+* Written as Markdown text documents.
+* Stored and tracked in a Git repo.
+* Proposed through GitHub PRs.
+* Reviewed and accepted/rejected through GitHub.
+
+The workflow for proposing and reviewing would then be the same as contributing
+any code or documentation to OpenTitan.
+
+### RFC file structure
+
+Having a repo of RFCs like this brings them closer to any other kind of
+documentation. To make best use of RFCs as documentation, this RFC also
+proposes adding some structure to RFCs:
+
+* RFCs are dated (and authored if necessary).
+* RFCs are based on some template file to help the author fully consider all
+  aspects of their proposal.
+* RFCs are assigned numeric IDs when accepted.
+* RFC numbers should start at 0001 and increase monotonically with date
+  accepted.
+* There should be no gaps in RFC numbers.
+
+Having RFCs as plain documentation lets us reference them when, for example:
+
+* Justifying the need for an issue or PR in OpenTitan.
+* Explaining why something was designed as it was in documentation.
+* Pointing to similar or conflicting RFCs to new proposals.
+
+### Private RFCs
+
+How and where private RFCs are stored is out of scope for this RFC, except to
+say that any redacted form of a private RFC that can be included must be. At the
+very least, private RFCs are still assigned the next RFC number when accepted.
+
+## Drawbacks
+
+* Some RFCs have private reasoning which cannot be submitted to a public repo.
+* RFC comments are still locked into GitHub's system, so are not portable.
+
+## Rationale and alternatives
+
+This workflow was chosen to reflect how we already contribute code and
+documentation to OpenTitan in an open source way. There should be almost no
+overhead to existing open source contributors who want to contribute RFCs.
+
+The choice to store RFCs as text files allows them to be used as any other
+documentation. Matching the RFC process to how we already contribute code and
+documentation makes the process more accessible. Text files can be written to
+and read by anybody, including people outside the opentitan.org domain.
+
+Opening up the RFC process makes our project governance more transparent and
+brings us closer inline with other open source projects. It shows that we don't
+gatekeep the project to corporate contributors or other insiders.
```

Comment:
Can we factually describe here what is being opened up, in a neutral language?

Reply:
Removed this, you're right that it wasn't neutral.